x-pack/plugins/cases/server/saved_object_types/migrations/comments.test.ts

```diff
@@ -0,0 +1,399 @@
+/*
```

Comment:
These tests mostly existed in the `index.test.ts` file, we've added some since 7.16/8.x all the same coverage should be there.